Auth0Tests/Generators.swift

```diff
@@ -121,10 +121,14 @@ func generateJWT(alg: String = JWTAlgorithm.rs256.rawValue,
     
     if let signature = signature {
         signaturePart = signature
+    } else {
+        var signingError: Unmanaged<CFError>?
```

Comment:
Same as above, remove the error variable if it is not going to be used and just pass `nil` instead.